src/lib/swapper/swappers/ArbitrumBridgeSwapper/ArbitrumBridgeSwapper.ts

```diff
@@ -0,0 +1,37 @@
+import type { AssetId } from '@shapeshiftoss/caip'
+import { arbitrumChainId, ethChainId } from '@shapeshiftoss/caip'
+import type { BuyAssetBySellIdInput, Swapper } from '@shapeshiftoss/swapper'
+import type { Asset } from '@shapeshiftoss/types'
+import { executeEvmTransaction } from 'lib/utils/evm'
+
+import { ARBITRUM_BRIDGE_SUPPORTED_CHAIN_IDS } from './utils/constants'
+
+export const arbitrumBridgeSwapper: Swapper = {
+  executeEvmTransaction,
+
+  filterAssetIdsBySellable: (assets: Asset[]): Promise<AssetId[]> => {
+    return Promise.resolve(
+      assets
+        .filter(asset => ARBITRUM_BRIDGE_SUPPORTED_CHAIN_IDS.sell.includes(asset.chainId))
+        .map(asset => asset.assetId),
+    )
+  },
+
+  filterBuyAssetsBySellAssetId: (input: BuyAssetBySellIdInput): Promise<AssetId[]> => {
+    const supportedChainIds = (() => {
+      switch (input.sellAsset.chainId) {
+        case ethChainId:
+          return [arbitrumChainId]
+        case arbitrumChainId:
+          return [ethChainId]
+        default:
+          return []
+      }
+    })()
+    return Promise.resolve(
+      input.assets
+        .filter(asset => supportedChainIds.includes(asset.chainId))
+        .map(asset => asset.assetId),
+    )
+  },
+}
```

Comment:
Note this only really does basic checks - we don't have a definitive related assets list that's as exhaustive as Arbitrum's lists would be.
Actual "is bridged asset pair" checks are done in here:

https://github.com/shapeshift/web/blob/e4485a306000723bbbcc14fa771037d427970023/src/lib/swapper/swappers/ArbitrumBridgeSwapper/getTradeQuote/getTradeQuote.ts#L84-L100